testing/run-scripts/image_make.sh

```diff
@@ -51,7 +55,17 @@ TMP_DIR=`mktemp -d`
 
 cp -R ${BASH_SOURCE%/*}/../integration/test $TMP_DIR/test
 
-cat <<EOF > $TMP_DIR/Dockerfile
+if [ "$DOCKER" = true ]
+then
+  cat <<EOF > $TMP_DIR/Dockerfile
+FROM soycode/uproxy-zork
+
+RUN apt-get -qq update
+RUN apt-get -qq upgrade
```

Comment:
By "leave it as is" do you mean removing the apt-get update/upgrade from this and assuming unattended-upgrades will take care of things as needed? Basically it means the Dockerfile will just be pulling the existing uproxy-zork image.